tutorial/tutorial.thrift

```diff
@@ -113,7 +113,7 @@ struct Work {
  * Structs can also be exceptions, if they are nasty.
  */
 exception InvalidOperation {
-  1: i32 what,
+  1: i32 whatOp,
```

Comment:
no, it should not - as a rule: Thrift does not keep track of language specific keywords.
This decision was made some time ago mostly because of two reasons:
 - with current language and platform coverage list of such keywords would be very long and hard to maintain
 - someone using Thrift to connect Java with Python should not be bothered with keywords coming from C++ library specifics

Proper solution was proposed: each generator should provide list of keywords extending "core Thrift keywords" and when generating code should check identifiers against those keywords (by some generic mechanisms). That would require some refactoring of generators in compiler, yet maybe someday..